tests/cypress/support/commands.js

```diff
@@ -267,23 +271,27 @@ Cypress.Commands.add('createRectangle', (createRectangleParams) => {
 });
 
 Cypress.Commands.add('switchLabel', (labelName, objectType) => {
-    cy.get(`.cvat-${objectType}-popover-visible`).find('.ant-select-selection-item').click();
+    cy.get(`.cvat-${objectType}-popover-visible`)
+        .find('.ant-select-selection-item')
+        .click();
     cy.get('.ant-select-dropdown')
         .not('.ant-select-dropdown-hidden')
         .find(`.ant-select-item-option[title="${labelName}"]`)
         .click();
 });
 
 Cypress.Commands.add('checkObjectParameters', (objectParameters, objectType) => {
-    cy.get('.cvat-draw-shape-popover').should('be.hidden');
+    cy.get(`.cvat-draw-${objectType.toLowerCase()}-popover-visible`).should('not.exist');
+    cy.get(`.cvat-draw-${objectType.toLowerCase()}-popover`).should('be.hidden');
+    cy.get(`.cvat-draw-${objectType.toLowerCase()}-popover`).should('have.attr', 'style').and('include', 'pointer-events: none');
```

Comment:
These actions are not expected to be in command with name ``checkObjectParameters``

Reply:
Moved it to a separate command.